packages/editor/src/js/vue/components/personalized-blocks-list/personalized-blocks-list-component.js

```diff
@@ -29,16 +37,19 @@ const PersonalizedBlocksListComponent = Vue.component(
     methods: {
       fetchPersonalizedBlocks() {
         this.isLoading = true;
+        this.vm.personalizedBlocks([]);
         const groupId = this.vm?.metadata?.groupId;
 
         axios
-          .get(getPersonalizedBlocks(), { params: { groupId } })
+          .get(getPersonalizedBlocks(), {
+            params: { groupId, searchTerm: this.searchTerm },
+          })
           .then((response) => {
             this.vm.personalizedBlocks(
               response.data?.items.map(({ content, ...blockInformation }) => ({
                 ...content,
                 blockInformation,
-                id: "" // This line is crucial. The `viewModel.loadDefaultBlocks` function will override this `id` when the block is added to the email content. Without this placeholder, there could be issues when adding personalized blocks in emails.
```

Comment:
Is it intentional to delete this comment?

Reply:
nope my bad !!